config.toml

```diff
@@ -76,6 +76,11 @@ files = ["/usr/bin/runc"]
 error = "ErrGoMissingSymbols"
 files = ["/usr/bin/runc"]
 
+[[rpm.runc.ignore]]
+# See OCPBUGS-36541.
+error = "ErrLibcryptoSoMissing"
```

Comment:
This should be `ErrLibcryptoMissing` (which is a different one from `ErrLibcryptoSoMissing`, see https://github.com/openshift/check-payload/blob/main/internal/types/errors.go)